src/main/java/sirius/biz/storage/layer2/jdbc/RemoveDeletedBlobsLoop.java

```diff
@@ -35,7 +35,7 @@
     @Nonnull
     @Override
     public String getName() {
-        return "storage-layer2-delete";
+        return "storage-layer2-blob-delete";
```

Comment:
i think the config values for orchestration of the new loops is missing in the component-biz.conf

Reply:
why rename this - layer2 only operates on blobs

Reply:
maybe we should extract common parts of the Mongo and JDBC impl and put them into a common base class.